source/vibe/core/drivers/libasync.d

```diff
@@ -893,7 +893,7 @@ final class LibasyncManualEvent : ManualEvent {
 	bool amOwner()
 	{
 		import std.algorithm : countUntil;
-		if (s_eventWaiters.length <= m_instance) return false;
+		if (s_eventWaiters.length <= m_instance + 1) return false;
```

Comment:
Wouldn't this cause `amOwner` to always return `false` for the latest `ManualEvent` that was created?